bin/crawl

```diff
@@ -1,39 +1,48 @@
 #!/usr/bin/env ruby
 
-require "bundler/setup"
-require "browser_crawler"
-require "pry" if ENV['DEBUG'] == 'true'
+require 'bundler/setup'
+require 'browser_crawler'
 
 options = BrowserCrawler::Options.parse_args
 ENV.update(options.transform_keys(&:to_s).transform_values(&:to_s))
 
-if options[:screenshots_path]
-  `mkdir -p #{options[:screenshots_path]}` unless File.directory?(options[:screenshots_path])
+if options[:screenshots_path] && !File.directory?(options[:screenshots_path])
+  `mkdir -p #{options[:screenshots_path]}`
+end
+
+if options[:report_format] && !%w[yaml csv].include?(options[:report_format])
+  puts("Report format #{options[:report_format]} is not recognized." \
+           ' Please, choose correct format from available both `csv` or `yaml`.')
```

Comment:
`Please choose the correct format from the list of available formats: 'csv', 'yaml'`